web/themes/joinup/templates/content/node--custom-page--listing-item.html.twig

```diff
@@ -85,7 +85,7 @@
         {{ title_suffix }}
       </div>
       <div class="listing__field">
-        <p class="listing__teaser">{{ content.body|render|striptags('<a>')|raw }}</p>
+        <p class="listing__teaser">{{ content.body|render|striptags|raw }}</p>
```

Comment:
Now that we strip all the tags, we can remove `|render|striptags|raw` and just configure the "smart trimmed" formatter to remove html tags. All these fields are using this formatter.